python/lsst/ts/ess/csc/ess_csc.py

```diff
@@ -94,9 +88,9 @@ async def _read_loop(self) -> None:
         try:
             while True:
                 data = await self.read()
-                if Key.RESPONSE in data:
-                    response = data[Key.RESPONSE]
-                    if response != ResponseCode.OK:
+                if common.Key.RESPONSE in data:
```

Comment:
An existing comment, but I am quite uneasy about having to search all of data to look for a key. The problem is what if a value shows up as some integer that isn't the message type?

It would be safer and more efficient to have a known field whose value was the kind of response, e.g. "id" or "msg_type".

I realize such a change should probably be done on a separate ticket.

Reply:
I created DM-31416 for this.